ludwig/data/dataframe/dask.py

```diff
@@ -224,20 +235,9 @@ def write_predictions(self, df: dd.DataFrame, path: str):
 
         ds = self.to_ray_dataset(df)
 
-        def to_tensors(batch: pd.DataFrame) -> pd.DataFrame:
-            data = {}
-            for c in batch.columns:
-                try:
-                    data[c] = TensorArray(batch[c])
-                except TypeError:
-                    # Not a tensor, likely a string which pyarrow can handle natively
-                    pass
-            return pd.DataFrame(data)
-
-        ds = ds.map_batches(to_tensors)
-
-        fs, path = get_fs_and_path(path)
-        ds.write_parquet(path, filesystem=PyFileSystem(FSSpecHandler(fs)))
+        with tensor_extension_casting(False):
```

Comment:
Why is this hard-coded to False?

Reply:
Similar to the below comment– I think it makes sense to have a single defined behavior when writing out to Parquet. The rationale here is that, since this function exports the data out of Ray Datasets into Parquet, there is no need to cast the data into TensorDtype before writing out to file. 

Are there particular conditions in which we might want to act otherwise? 

Regardless, it probably makes sense to have a comment here– I'll be sure to add that.